Fix for support request-system-reboot

chidanandpujar opened the pull request:
Fix for issue https://github.com/Juniper/ansible-junos-stdlib/issues/725

dineshbaburam91 reviewed (changes requested):
Please validate the same in Dual RE EVO using ansible playbook in both local and PyEZ connection

chidanandpujar commented:
> Please validate the same in Dual RE EVO using ansible playbook in both local and PyEZ connection

Please find the Dual RE reboot logs.
```
(venv) root@masterhost:~/ansible_release_test1/ansible-junos-stdlib/tests# ansible-playbook -i inventory  pb.juniper_junos_system.yml 
[DEPRECATION WARNING]: [defaults]collections_paths option. Reason: does not fit var naming standard, use the singular form collections_path instead 
Alternatives: none. This feature will be removed in version 2.19. Deprecation warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.

PLAY [Test juniper.device.system module] ********************************************************************************************************************

TASK [Reboot all REs of the device] *************************************************************************************************************************
changed: [local_connection_testcases]

TASK [CHECK TEST 1] *****************************************************************************************************************************************
ok: [local_connection_testcases] => {
    "changed": false,
    "msg": "All assertions passed"
}

TASK [Checking NETCONF connectivity] ************************************************************************************************************************
ok: [local_connection_testcases]

PLAY RECAP **************************************************************************************************************************************************
local_connection_testcases : ok=3    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   


[vrf:none] root@evoeventtestx-RE0-re0:~#                                                                                
*** System shutdown message from root@evoeventtestx-RE0-re0 ***                
                                                                               
                                                                               
                                                                               
                                                                               
System going down IMMEDIATELY                                                  
                                                                               
                                                                               
                                                                               
Connection to 10.53.106.244 closed by remote host.
Connection to 10.53.106.244 closed.
chipujar@q-pod27-vmm:~/IMAGES$ 


(venv) root@masterhost:~/ansible_release_test1/ansible-junos-stdlib/tests# ansible-playbook -i inventory  pb.juniper_junos_system.yml 
[DEPRECATION WARNING]: [defaults]collections_paths option. Reason: does not fit var naming standard, use the singular form collections_path instead 
Alternatives: none. This feature will be removed in version 2.19. Deprecation warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.

PLAY [Test juniper.device.system module] ********************************************************************************************************************

TASK [Reboot all REs of the device] *************************************************************************************************************************
changed: [pyez_connection_testcases]

TASK [CHECK TEST 1] *****************************************************************************************************************************************
ok: [pyez_connection_testcases] => {
    "changed": false,
    "msg": "All assertions passed"
}

TASK [Checking NETCONF connectivity] ************************************************************************************************************************
ok: [pyez_connection_testcases]

PLAY RECAP **************************************************************************************************************************************************
pyez_connection_testcases  : ok=3    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   

chipujar@q-pod27-vmm:~/IMAGES$ ssh root@10.53.106.244
Password: 
--- JUNOS 22.3R3.8-EVO Linux (none) 5.2.60-yocto-standard-g278f9ad #1 SMP PREEMPT Mon May 29 06:00:28 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
[vrf:none] root@evoeventtestx-RE0-re0:~#                                                                                
*** System shutdown message from root@evoeventtestx-RE0-re0 ***                
                                                                               
                                                                               
                                                                               
                                                                               
System going down IMMEDIATELY                                                  
                                                                               
                                                                               
                                                                               
Connection to 10.53.106.244 closed by remote host.
Connection to 10.53.106.244 closed.
chipujar@q-pod27-vmm:~/IMAGES$ 
```

Thanks
Chidanand
